source/indexes/atlas-search-index.txt

```diff
@@ -0,0 +1,137 @@
+
+The following sections provide code examples that demonstrate how to use
+each of the preceding methods.
+
+.. _scala-atlas-search-index-create:
+
+Create a Search Index
+---------------------
+
+You can use the ``createSearchIndex()`` and the ``createSearchIndexes()`` methods to
+create one or more Atlas Search indexes.
+
+You can also use these methods to create Atlas Vector Search Indexes.
+Atlas Vector Search enables you to perform semantic searches on vector
+embeddings stored in MongoDB Atlas. To learn more about this feature,
+see the :atlas:`Atlas Vector Search Overview
+</atlas-vector-search/vector-search-overview/>`.
+
+The following code example shows how to create an Atlas Search index:
+
+.. literalinclude:: /includes/indexes/atlas-search-index.scala
+   :language: scala
+   :start-after: start-create-search-index
+   :end-before: end-create-search-index
+   :dedent:
+
+The following code example shows how to create multiple indexes:
+
+.. literalinclude:: /includes/indexes/atlas-search-index.scala
+   :language: scala
+   :start-after: start-create-search-indexes
+   :end-before: end-create-search-indexes
+   :dedent:
+
+To learn more about the syntax used to define Atlas Search indexes, see the
+:atlas:`Review Atlas Search Index Syntax </atlas-search/index-definitions>` guide
+in the Atlas manual.
+
+.. _scala-atlas-search-index-list:
+
+List Search Indexes
+-------------------
+
+You can use the ``listSearchIndexes()`` method to return all Atlas Search indexes in a
+collection.
+
+The following code example shows how to print a list of the search indexes in
+a collection:
+
+.. literalinclude:: /includes/indexes/atlas-search-index.scala
+   :language: scala
+   :start-after: start-list-search-indexes
+   :end-before: end-list-search-indexes
+   :dedent:
+
+.. _scala-atlas-search-index-update:
+
+Update a Search Index
+---------------------
+
+You can use the ``updateSearchIndex()`` method to update an Atlas Search index.
+
+The following code shows how to update a search index:
+
+.. literalinclude:: /includes/indexes/atlas-search-index.scala
+   :language: scala
+   :start-after: start-update-search-indexes
+   :end-before: end-update-search-indexes
+   :dedent:
+
+.. _scala-atlas-search-index-drop:
+
+Delete a Search Index
+---------------------
+
+You can use the ``dropSearchIndex()`` method to delete an Atlas Search index.
+
+The following code shows how to delete a search index from a collection:
+
+.. literalinclude:: /includes/indexes/atlas-search-index.scala
+   :language: scala
+   :start-after: start-drop-search-index
+   :end-before: end-drop-search-index
+   :dedent:
+
+Additional Information
+----------------------
+
+To learn more about MongoDB Atlas Search, see the :atlas:`Atlas Search Indexes </atlas-search/atlas-search-overview/>`
+documentation.
+
+API Documentation
+~~~~~~~~~~~~~~~~~
+
+To learn more about the types mentioned in this guide, see the following API documentation:
```

Comment:
As far as I can tell, there are no docs for the individual methods above in neither the Scala docs nor the Java core docs.